src/ConfigEditor/ConfigEditor.test.tsx

```diff
@@ -209,21 +209,27 @@ describe('ConfigEditor', () => {
         {...props}
         options={{
           ...props.options,
-          jsonData: { ...props.options.jsonData, database: 'test-db' },
+          jsonData: { ...props.options.jsonData, database: 'test-db', useManagedSecret: false },
         }}
         onOptionsChange={onChange}
       />
     );
 
-    const selectEl = screen.getByLabelText(selectors.components.ConfigEditor.ClusterID.input);
+    const selectEl = screen.getByRole('combobox', { name: selectors.components.ConfigEditor.ClusterID.input });
```

Comment:
I think the old test was selecting the wrong Cluster identifier input. As it was set up, cluser Id Select shouldn't have been shown, instead the cluster disabled autofilled field should have. 
I added a prop to get it to show, and a selector for the combobox in particular (not the input).
I could be getting this wrong, but I think it should be ok now. I double checked [this PR](https://github.com/grafana/redshift-datasource/pull/228) to verify the logic.